src/commands/run.js

```diff
@@ -28,7 +28,7 @@ export const handler = async (argv) => {
     const jobName = getJobName(argv)
     const buildName = argv.build || `${jobName} - ${(new Date()).toString()}`
     const metrics = getMetricParams(argv)
-
+    const crmuxdriverVersion = argv.crmuxdriverVersion || 'stable'
```

Comment:
Given that this is not defined in the [`RUN_CLI_PARAMS `](https://github.com/saucelabs/speedo/blob/master/src/constants.js#L76) I wonder if this will be picked up. Also it would be nice to document this in the readme.